docs/advanced/longhorn-v2.md

```diff
@@ -34,6 +34,7 @@ The Longhorn V2 Data Engine currently does not support the following operations:
 
 - Backing image creation and usage
```

Comment:
We can remove this, since we adopt PVC for image in 3rd party storage for v2.

Reply:
Good catch, although it might be worth doing that in a separate PR, because I think there's other things further down that will also need changing for v1.5 using 3rd party store (e.g. the current limitation on boot disks being V1 only)